app/providers/backup.tsx

```diff
@@ -34,7 +35,7 @@ export interface ContextActions {
   addBackupJob: (space: SpaceDID, chats: Set<bigint>, period: Period) => Promise<JobID>
   removeBackupJob: (job: JobID) => Promise<void>
   // setBackup: (id: Link|null) => void
-  // setDialog: (id: bigint|null) => void
+  setDialog: (dialog: Dialog | null) => void
```

Comment:
Hmm, the idea here was that you'd set the dialog ID and the provider would take care of loading the dialog details and messages, exposing them in the reactive variables `dialog` and `messages`.